base/float.jl

```diff
@@ -386,10 +386,35 @@ end
 end
 
 # fused multiply-add
-fma_libm(x::Float32, y::Float32, z::Float32) =
-    ccall(("fmaf", libm_name), Float32, (Float32,Float32,Float32), x, y, z)
-fma_libm(x::Float64, y::Float64, z::Float64) =
-    ccall(("fma", libm_name), Float64, (Float64,Float64,Float64), x, y, z)
+function fma_libm(x::Float32, y::Float32, z::Float32)
+    r = ccall(("fmaf", libm_name), Float32, (Float32,Float32,Float32), x, y, z)
+    # This is necessary on 32-bit Intel Linux, probably on OS X, and
+    # maybe on Windows
+    # TODO: Disable this for non-Intel systems
+    if is(Int, Int32)
```

Comment:
Looks like this is what we need: http://llvm.org/docs/doxygen/html/classllvm_1_1Triple.html#a547abd13f7a3c063aa72c8192a868154

The Triple is already used here: https://github.com/JuliaLang/julia/blob/master/src/codegen.cpp#L410